ArduCopter/radio.cpp

```diff
@@ -115,7 +112,7 @@ void Copter::read_radio()
     }else{
         uint32_t elapsed = tnow_ms - last_radio_update_ms;
         // turn on throttle failsafe if no update from the RC Radio for 500ms or 2000ms if we are using RC_OVERRIDE
-        if (((!failsafe.rc_override_active && (elapsed >= FS_RADIO_TIMEOUT_MS)) || (failsafe.rc_override_active && (elapsed >= FS_RADIO_RC_OVERRIDE_TIMEOUT_MS))) &&
+        if (((!RC_Channels::has_active_overrides() && (elapsed >= FS_RADIO_TIMEOUT_MS)) || (RC_Channels::has_active_overrides() && (elapsed >= FS_RADIO_RC_OVERRIDE_TIMEOUT_MS))) &&
```

Comment:
Probably worth stashing this in a local bool rather then calling it twice, it's a fairly expensive call (grab the singleton + iterates all the channels).

Reply:
Realised we can not call has_active_overrides in the we've-got-radio case, too.  We only need to know if there are overrides when determining whether the RC receiver is present - which we've decided is always-present-if-ever-present.